science/hacktv/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        kesterlester hacktv b3d9ecb87f042bb6729824fabb40d55d145c3e65
```

Comment:
no, this should use upstream repository not your fork of it

Reply:
yes, I know, I'm not there yet. I'm working in a private rep. I'll push here when I 'm done.  I'm fixing things peacemeal.  Don't review again until I say it's ready! :)

Reply:
you can always request upstream to at least tag a version or ideally even start doing releases

Reply:
> yes, I know, I'm not there yet. I'm working in a private rep. I'll push here when I 'm done. I'm fixing things peacemeal. Don't review again until I say it's ready! :)

okay, I'll mark this as draft then. Once you're done with it you can set it to "ready for review"

Reply:
> > yes, I know, I'm not there yet. I'm working in a private rep. 

FYI you're working in your fork but as soon as you push there this PR will get updated as well

Reply:
Well, I'm working in two places, a private rep AND my fork.  But that aside, I see you marked this rep as "work in progress".  I didn't know that possibility existed. I'm hoping that with that enablement (thank you!) then you and others should not be bothered anymore .... even if I do push to my fork .... until such time as I mark things no longer "in progress".

Is that correct?

I don't want to cause you (or others) unnecessary noise.

Reply:
thank you